server/.eslintrc.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     tsconfigRootDir : __dirname, 
     sourceType: 'module',
   },
-  plugins: ['@typescript-eslint/eslint-plugin', 'import', 'unused-imports'],
+  plugins: ['@typescript-eslint/eslint-plugin', 'import', 'unused-imports','twenty-ts'],
```

Comment:
You'll need to add the eslint-plugin-twenty-ts to server package.json and to add its installation to server yarn install command (similar to what has been done in front/package.json)
I suspect this might be not straightforward + you'll need to fix many linter errors, so let's do that in another PR :)

Reply:
I am going fix those now. Thanks for your review.

Reply:
Yes, let's fix all the linters errors in another PR.